lib/ups/builders/address_builder.rb

```diff
@@ -128,6 +128,7 @@ def to_xml
           # By setting the residential address indicator tag.
           # We can trigger UPS to check for any surcharge related to that area
           # and apply the charge automically.
+          # ** WARNING: The usage of an empty flag is not inlcuded in UPS's documentation, subject to future changes.
```

Comment:
the documentation states this is an empty tag, check page 24 of the developer guide

Reply:
Nice catch! Removed that warning from PR